cmd/root.go

```diff
@@ -20,17 +20,18 @@ import (
 // invoked during the execution of the CLI. It also is responsible for managing the output configuration as well as the
 // output signal itself, which will be written after the execution of the invoked command's Run function.
 type WebScan struct {
-	Version      string
-	RootFlags    config.RootFlags
-	OutputConfig writer.OutputConfig
-	OutputSignal signal.Signal
-	RootCmd      *cobra.Command
-	VersionCmd   *cobra.Command
-	FuzzCmd      *cobra.Command
-	ProbeCmd     *cobra.Command
-	SpiderCmd    *cobra.Command
-	VulnCmd      *cobra.Command
-	AppCmd       *cobra.Command
+	Version         string
+	RootFlags       config.RootFlags
+	OutputConfig    writer.OutputConfig
+	OutputSignal    signal.Signal
+	RootCmd         *cobra.Command
+	VersionCmd      *cobra.Command
```

Comment:
Not a blocker but we don't actually need to store each command on the struct. We can just attach them to the root command and only store the root.

Reply:
ack